extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -865,6 +865,7 @@ export class AmpA4A extends AMP.BaseElement {
   viewportCallback(inViewport) {
     if (this.friendlyIframeEmbed_) {
       setFriendlyIframeEmbedVisible(this.friendlyIframeEmbed_, inViewport);
+      this.friendlyIframeEmbed_.setCssAnimPlayingState(inViewport);
```

Comment:
Sounds good. I don't think it's necessary to add it to the observable since the logic cannot be decoupled while inside the same class.